client/app/app.js

```diff
@@ -36,7 +36,28 @@ app.config(function($stateProvider, $urlRouterProvider, $httpProvider) {
         controller: 'HospitalCtrl',
         templateUrl: 'app/hospitalMap/hospital-map.html'
       });
+      
+      $httpProvider.interceptors.push('AttachTokens');
 })
-
+.controller('NavigationBarController', function ($scope, Auth) {
```

Comment:
Adding a nav bar controller so we can determine whether to hide/show different things